### DISCUSSION TITLE
Filter workflow runs by workflow

### DISCUSSION THREAD

**Discussion by cunla** [community/community#70753] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I am developing [a JetBrains plugin to manage github actions from the IDEs (IntelliJ, PyCharm, etc.)](https://github.com/cunla/ghactions-manager).
There have been several requests for a feature to filter workflow runs by workflow - however, since it is not supported in the [List workflows REST API](https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#list-repository-workflows), it is difficult to achieve it.

Would it be possible to add this feature to the REST API?

**Comment by ghostinhershell** [comment]
Hi there @cunla, 

Thanks so much for reaching out in our community. I can file a feature request on your behalf for this feature 🚀

**Comment by Pieeer1** [comment]
Seconding this feature. Either that or the ability to filter by workflow name (regex or some other search) would be ideal
